benchmarking/definitions/definition_nasbench201.py

```diff
@@ -1,110 +0,0 @@
-# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License").
-# You may not use this file except in compliance with the License.
-# A copy of the License is located at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# or in the "license" file accompanying this file. This file is distributed
-# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
-# express or implied. See the License for the specific language governing
-# permissions and limitations under the License.
-from syne_tune.config_space import choice
-from syne_tune.blackbox_repository.conversion_scripts.scripts.nasbench201_import import (
-    CONFIG_KEYS,
-    METRIC_VALID_ERROR,
-    RESOURCE_ATTR,
-    BLACKBOX_NAME,
-)
-
-
-DATASET_NAMES = [
-    "cifar10",
-    "cifar100",
-    "ImageNet16-120",
-]
-
-
-METRIC_ELAPSED_TIME = "metric_elapsed_time"
-
-
-# First is default value
-x_range = ["skip_connect", "none", "nor_conv_1x1", "nor_conv_3x3", "avg_pool_3x3"]
-
-
-_config_space = {k: choice(x_range) for k in CONFIG_KEYS}
-
-
-def nasbench201_default_params(params=None):
```

Comment:
I think when we remove this some docs/examples might fail, same for `nasbench201_benchmark()`.

For example
`https://github.com/awslabs/syne-tune/blob/prepare_sphinx2/docs/tutorials/pasha/scripts/launch_pasha_nasbench201.py#L9`
`https://github.com/awslabs/syne-tune/blob/prepare_sphinx2/docs/tutorials/pasha/scripts/launch_pasha_nasbench201_study.py#L22`

I didnt find any more but maybe I was not good enough at looking

Reply:
Yes, you are right. The PASHA tutorial is defunct. But it has been defunct before already. Note that it does not feature in our hosted docs.